packages/core/src/external/carequality/ihe-gateway-v2/outbound/__tests__/constants.ts

```diff
@@ -219,7 +219,7 @@ export const expectedDqDocumentReference = [
     language: "en-US",
     size: 141255,
     title: "Continuity of Care Document",
-    creation: "2024-03-27T21:43:04.878Z",
+    creation: "2024-03-27T21:43:04",
```

Comment:
This is a valid ISO date, shouldn't we have at least one example with full ISO date?